docker/prepare_proxy_test.sh

```diff
@@ -1,37 +1,41 @@
-#!/bin/sh
+#!/bin/bash
+# in glcoud: prepare_proxy_test.sh 
+# in your dev environment: HUB=quay.io/${whoami} prepare_proxy_test.sh local
+#
 
 set -o errexit
 set -o nounset
 set -o pipefail
 set -ex
 
 # customize for development / debug setup
-HUB=docker.io/$(whoami)
+# by specifying an environment var HUB to desired setting
+if [ "$HUB" = "" ]; then
+    HUB=docker.io/$(whoami)
+fi
 TAG=test
 NAMESPACE=prepare-proxy-test0
 
-PLATFORM=gcloud
-GCLOUD_CLUSTER_NAME=c1
-GCLOUD_CLUSTER_ZONE=us-central1-a
+LIST_CIDRS=""
```

Comment:
It would be better to either rely on ENVVAR or add proper flags support (e.g. getopt) rather than rely on implicit ordering to set parameters.